app/src/main/java/tech/relaycorp/gateway/data/preference/PublicGatewayPreferences.kt

```diff
@@ -65,6 +65,6 @@ class PublicGatewayPreferences
     companion object {
         @VisibleForTesting
         internal const val DEFAULT_ADDRESS = "https://poweb-frankfurt.relaycorp.cloud"
-        private const val COGRPC_ADDRESS = "https://cogrpc-frankfurt.relaycorp.cloud"
+        private const val COGRPC_ADDRESS = "https://frankfurt.relaycorp.cloud"
```

Comment:
`DEFAULT_ADDRESS` will become `"https://frankfurt.relaycorp.cloud"` in a future PR too, so when we get there I'll delete `COGRPC_ADDRESS`.